archeo/inc/patterns/image-with-headline-description.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Image with headline and description
+ */
+return array(
+	'title'      => __( 'Image with headline and description', 'archeo' ),
+	'categories' => array( 'pages' ),
+	'content'    => '<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"5vw","right":"5vw","bottom":"0","left":"5vw"}}}} -->
+	<div class="wp-block-group alignfull" style="padding-top:5vw;padding-right:5vw;padding-bottom:0;padding-left:5vw"><!-- wp:media-text {"mediaPosition":"right","mediaLink":"' . esc_url( get_template_directory_uri() ) . '/assets/images/chahk.jpg","mediaType":"image","mediaWidth":64,"verticalAlignment":"top"} -->
+	<div class="wp-block-media-text alignwide has-media-on-the-right is-stacked-on-mobile is-vertically-aligned-top" style="grid-template-columns:auto 64%"><figure class="wp-block-media-text__media"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/chahk.jpg" alt="' . esc_attr__( 'Chahk: rain deity', 'archeo' ) . '"/></figure><div class="wp-block-media-text__content"><!-- wp:heading {"style":{"typography":{"fontSize":"clamp(64px, 6vw, 100px)","lineHeight":"1"},"spacing":{"margin":{"bottom":"48px"}}}} --><h2 id="chahk-raindeity" style="font-size:clamp(64px, 6vw, 100px);line-height:1;margin-bottom:48px">' . wp_kses_post( __( 'CHAHK:<br>RAIN<br>DEITY', 'archeo' ) ) . '</h2>
```

Comment:
I have a couple questions:

- Should the font-size value be a custom variable in theme.json too? 
- Semantically, is it okay that this is an h2 instead of a paragraph? I changed it to a heading so that I could get margin control...

Reply:
> Should the font-size value be a custom variable in theme.json too?

I have the same question, and I think it would be great if it was a variable, I just don't know where it should sit in theme.json (Maybe just an extra font size? But then I don't think it's selectable from the editor.)

> Semantically, is it okay that this is an h2 instead of a paragraph?

I think this is OK, as it's acting as a heading for this pattern.